zerver/tests/test_users.py

```diff
@@ -393,7 +393,7 @@ def test_cache_behavior(self):
             with simulated_empty_cache() as cache_queries:
                 user_profile = self.example_user('hamlet')
 
-        self.assert_length(queries, 1)
+        self.assert_length(queries, 2)
```

Comment:
can you explain why the various query counts went up with this change?  I don't see why it should.